module/solutions/helper.js

```diff
@@ -178,6 +178,9 @@ module.exports = class SolutionsHelper {
         if(checkDate){
           if(solutionData.hasOwnProperty("endDate")){
             solutionData.endDate = gen.utils.getStartDate(solutionData.endDate)
+            if(solutionData.endDate > programData[0].endDate){
+              solutionData.endDate = programData[0].endDate
```

Comment:
@ankitpws we should add it in update solution as well right?

Reply:
yes @aks30